src/content/posts/2025-video-chapter-request.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: "🎬 Lights, Camera, PyLadies! Share Your Chapter Video for PyLadiesCon 2025 💜"
+meta_title: "🎬 Lights, Camera, PyLadies! Share Your Chapter Video for PyLadiesCon 2025 💜"
+description: "We’re thrilled to invite all PyLadies chapters around the world to take part in something special for the **third edition of PyLadiesCon. 🌎✨"
+date: 2025-10-08T05:00:00Z
+image: "/images/posts/video-chapter-request.png"
+categories: ["Blog Post",]
+authors: ["Denny Pérez"]
+tags: ["python", "community", "conference", "pyladies"]
+draft: false
+---
+
+Hello PyLadies Community\!
+
+We’re thrilled to invite all **PyLadies chapters around the world** to take part in something special for the **third edition of PyLadiesCon**, happening **December 5–7, 2025**\! 🌎✨
+
+Each year, we showcase short videos from our chapters to highlight the incredible work happening across our global community — and we want **your chapter** to be part of it\!
+
+## **💡 What’s this about?**
+
+We’d love a **90-second (1 minute 30 seconds)** video introducing your PyLadies chapter — a friendly way to say “hi” to the global audience and share a glimpse of your journey, your community spirit, and your impact.
+
+Your video will be featured during PyLadiesCon and celebrated across our social channels — it’s a great opportunity to show the world how your chapter empowers women and allies in Python and open source\! 🐍💪
+
+## **🎥 What to include in your video**
+
+Here are some ideas to guide you:
+
+* **Chapter Introduction:** Tell us your chapter name, city, and country 🌍
+
+* **Organizers:** Introduce your amazing organizers (names \+ roles) 💬
+
+* **Your Story:** What inspired your chapter? What’s your mission? ❤️
+
+* **Community Highlights:** Mention a few events, projects, or accomplishments you’re proud of 🎉
+
+* **How to Connect:** Share your social media links or website so others can find you 🔗
+
+✨ *Pro tip:* Prepare a short script before recording — it helps keep your message clear and confident.
+
+If you’d like, you can use our [**PyLadiesCon slides template**](https://docs.google.com/presentation/d/1AfuoxJEMRfVJ648Il5BuDNMGHrKhvreSOfsMRtRqWGM/edit?usp=sharing) for your visuals, or simply add the **\[PyLadiesCon logo\]** to your existing materials.
```

Comment:
Regarding the "add the PyLadiesCon logo", do we have the link to the logo? Currently this is shown as a plain text.